doc/UserGuide.md

```diff
@@ -53,6 +53,14 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorting all persons' name in alphabetical order : `sortname`
```

Comment:
Is it better to just create one sort command and add name/email as parameters? I think this will make it easier for users to remember :)